bank.py

```diff
@@ -1,19 +1,21 @@
 from user import User
 from typing import Optional
-
+import json
 
 class ATM:
     def __init__(self):
         self.users = []
+        self.users_JSON = []
```

Comment:
why do we need this?
u use it only inside one method. maybe move it to this method?